tests/Cms/System/SystemTest.php

```diff
@@ -507,6 +502,22 @@ public function testIsOk()
 		$this->assertTrue($system->isOk());
 	}
 
+	/**
+	 * @covers ::isOk
+	 */
+	public function testIsOkContentMissingPermissions()
+	{
+		$app    = $this->app->clone([]);
+		$system = new System($app);
+
+		$before = fileperms($app->root('content'));
+		chmod($app->root('content'), 0000);
+
+		$this->assertFalse($system->isOk());
+
+		chmod($app->root('content'), $before);
```

Comment:
This should happen in a `tearDown()` method, otherwise the test suite will error out on cleanup if the assertion fails and this is not run.

Reply:
Without it a number of tests started failing.

Reply:
@lukasbestle could you please give it a try to move it to the `tearDown()` method? My attempts just end in phpunit completely blowing up with errors.

Reply:
@lukasbestle of course 💡 That explanation makes total sense.